catboost/libs/quantized_pool_analysis/quantized_pool_analysis.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <catboost/libs/algo/apply.h>
+#include <catboost/libs/algo/index_calcer.h>
+#include <catboost/libs/data_new/quantization.h>
+#include <catboost/libs/data_new/data_provider.h>
+#include <catboost/libs/data_new/data_provider_builders.h>
+#include <catboost/libs/model/model.h>
+#include <catboost/libs/model/formula_evaluator.h>
+#include <catboost/libs/target/data_providers.h>
+
+#include <util/generic/vector.h>
+
+
+namespace NCB {
+
+    struct TBinarizedFloatFeatureStatistics {
+        TVector<float> Borders;
+        TVector<ui8> BinarizedFeature;
+        TVector<float> MeanTarget;
+        TVector<float> MeanPrediction;
+        TVector<size_t> ObjectsPerBin;
+        TVector<double> PredictionsOnVaryingFeature;
+    };
+
+    struct TBinarizedOneHotFeatureStatistics{
```

Comment:
Let's unite TBinarizedFloatFeatureStatistics and TBinarizedOneHotFeatureStatistics:
```
struct TFeatureStatistics {
    TVector<float> Borders; // Maybe empty in case of one hot features.
    TVector<int> BinarizedFeature;
    TVector<float> MeanTarget;
    TVector<float> MeanPrediction;
    TVector<size_t> ObjectsPerBin;
    TVector<double> PredictionsOnVaryingFeature;
};
```